start.py

```diff
@@ -298,6 +301,11 @@ def cm_server_not_dead(primary_node):
     logger.info('Deploying client config ...')
     cm_cluster.deploy_client_config()
 
+    # This is needed due to the fact spark2 service runs first_run_service command while configuring.
```

Comment:
Do you really wanna stop all services like this even if spark2 isn't present? Should this be conditional on that?